README.md

```diff
@@ -1 +1,76 @@
 # mirador-keyboardnavigation
+
+[![npm package][npm-badge]][npm]
+[![required Mirador version][mirador-badge]][mirador]
+
+A Mirador 3 plugin which adds keyboard shortcuts for the following events:
+
+- scroll to the first canvas (default shortcut: `ctrl+left`)
+- scroll to the previous canvas (default shortcut: `left`)
+- scroll to the next canvas (default shortcuts: `right`, `space`)
+- scroll to the last canvas (default shortcut: `ctrl+right`)
+- toggle fullscreen mode (default shortcut: `enter`)
+
+## Installation
+
+Currently the plugin can only be used if you build your own Mirador JavaScript bundle.
+To include the plugin in your Mirador installation, you need to install it
+from npm with `npm install mirador-keyboardnavigation`, import it into your project
+and pass it to Mirador when you instantiate the viewer:
+
+```javascript
+import Mirador from 'mirador/dist/es/src/index';
+import keyboardNavigationPlugin from 'mirador-keyboardnavigation/es';
+
+const miradorConfig = {
+  // Your Mirador configuration
+}
+Mirador.viewer(config, [...keyboardNavigationPlugin]);
+```
+
+## Configuration
+
+You can configure the plugin just globally by adding the `keyboardNavigation` entry to
+the top-level configuration:
+
+```javascript
+const miradorConfig = {
+  // ...
+  keyboardNavigation: {
+    // Global config, see available settings below
```

Comment:
Might be a good idea to have two or three examples in here to make it more easily understandable.

Reply:
I added some examples.